about.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: page
+title: About
+permalink: /about
+---
+
+Austin Propositions is a site written and maintained by a small group of
+Austinites who got tired of being confused by ballot language of propositions
+that go to the whole city in elections. They decided to do research on each one
+and share what they found with anyone who would find it useful.
+
+The current authors are:
+* [Brad Fults](https://bradfults.com)
+* [Ben Hamill](http://benhamill.com)
+* [James Martin](#)
+* [Jimmy Lyke](#)
```

Comment:
@jameslmartin @jlyke: What would y'all wanna link to? I feel like "Facebook" or even "Twitter" would be suboptimal, but I'm willing to accept the idea that I'm a fringe radical in that respect. 😜

Reply:
OTOH, we could write a little 2-sentence bio about each of us and link or not link from there individually. IDK which is better.